src/codeu/chat/client/commandline/Chat.java

```diff
@@ -323,7 +346,18 @@ public void invoke(List<String> args) {
           } else {
             panels.push(createConversationPanel(conversation));
 
-            //command convo-id (uuid of)convo-owner convo-title creation-time
+            // If this user isn't in the new conversations map, add them with a new Set
+            Set<Uuid> newConversations;
```

Comment:
You can use `computeIfAbsent`: https://docs.oracle.com/javase/8/docs/api/java/util/Map.html#computeIfAbsent-K-java.util.function.Function-

`Set<Uuid> newConversations = newConversationsMap.computeIfAbsent(user.user.id, id -> new HashSet<>());`